src/raw.rs

```diff
@@ -7,22 +7,20 @@
 //
 
 use std;
-use nom;
+use std::result::Result::Err;
```

Comment:
I don't think this should be required - I think this is imported by the default prelude.  `Err(e)` has always just worked for me without a `use` statement, anyway.

Just to be sure, you should verify that `cargo check` and `cargo clippy` both come up clean.